Components/Coroutines/Town/TrinityTownRun.SellItems.cs

```diff
@@ -56,24 +56,28 @@ public static async Task<bool> SellItems()
                 if (merchant == null)
                 {
                     s_logger.Error($"[{nameof(SellItems)}] Unable to find merchant info for this area :(");
-                    return true;
+                    return CoroutineResult.Failed;
                 }
 
-                if (!await CommonCoroutines.MoveAndInteract(merchant.GetActor(), () => UIElements.VendorWindow.IsVisible))
-                    return false;
+                if (await CommonCoroutines.MoveAndInteract(
+                        merchant.GetActor(),
+                        () => UIElements.VendorWindow.IsVisible) == CoroutineResult.Running)
```

Comment:
Just for the sake of cleaner explanation for my previous comment, we continue execution here when MoveAndInteract returns Failure. Code after this will fail miserably in this case.